package.json

```
@@ -60,7 +60,7 @@
     "test:e2e:debug": "playwright test --project chromium --headed",
     "test:e2e:report": "npx playwright show-report",
     "test": "run-s lint test:unit:single test:e2e",
-    "update": "npm-check -u",
+    "update": "ncu -u",
```

If I am not mistaken, `ncu -u` only updates `package.json`, but it does not install the updated versions (and most importantly it does not update the lockfile). Should manually install:

```suggestion
    "update": "ncu -u && npm install",
```

---

Updated